### PR TITLE
Footer component

### PR DESCRIPTION
**Pull request by naremanhilles**
Relates: #21

### CODE REVIEW

**Review by Fatmasiam — APPROVED** [review]

File: Client/src/components/common/footer.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
```
Comment (kannanhassouna) [inline Client/src/components/common/footer.jsx:1]: remove unnecessary classes 
```suggestion
import React from 'react';
```

File: Client/src/components/common/footer.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+function Footer(){
+    return(
+      <div className="footer">
+     <i class="fab fa-facebook-f"></i>
```
Comment (kannanhassouna) [inline Client/src/components/common/footer.jsx:6]: Indentation !!!

**Review by kannanhassouna — CHANGES_REQUESTED** [review]
Where the style of component

**Comment by naremanhilles** [comment]
> Where the style of component

**Comment by naremanhilles** [comment]
style is abdullah's task.

**Pull request closed without merging**